### PR TITLE
Ch redesign about page 126584711

### PR DESCRIPTION
**Pull request by andela-cnwosu**
## Redesign of About Page

**What does this PR do?**
Redesign of About page and fixing of erratic test bugs

**Description of Task to be completed?**
The redesign is necessary to make the page responsive and to allow for dynamic generation of statistics about the site. Erratic feature test bugs were also fixed.

**How should this be manually tested?**
Clone repo on https://github.com/andela/eventx, start the server and visit page at localhost:3000

screencapture-localhost-8000-about-1474246280417
screencapture-localhost-8000-about-1474246309088

### CODE REVIEW

File: app/assets/stylesheets/welcome.scss.erb [hunk 1]
```diff
@@ -150,6 +157,72 @@ input[type=text]:focus:not([readonly]) {
...
+  text-align: center;
+  padding: 50px 0;
+  .bubble {
+    display: inline-block;
+    padding: 20px;
+    width: 280px;
+    height: 280px;
+    opacity: 0.7;
+    border-radius: 50%;
+  }
```
Comment (andela-megwali) [inline app/assets/stylesheets/welcome.scss.erb]: NITPICK: Press enter here.

File: app/assets/stylesheets/welcome.scss.erb [hunk 2]
```diff
@@ -306,15 +365,132 @@ nav.custom_nav {
   cursor: text;
 }
 
-.transform-text {
-  text-transform: capitalize !important;
+.transform-text { text-transform: capitalize !important; }
+
+
```
Comment (andela-megwali) [inline app/assets/stylesheets/welcome.scss.erb]: NITPICK: Is there a need for the extra line?

**Comment by andela-bmakinwa** [comment]
I like 💯

File: app/views/welcome/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    </div>
+  </div>
+  <div class="row">
+    <div class="col s12 m6 l4">
+      <div class="profile-card">
+        <div class="card-image">
+          <%= image_tag 'team_profile_4.jpg' %>
+          <div class="feature-details">
+            <div class="feature-title">
+              Michael Egwali
```
Comment (andela-megwali) [inline app/views/welcome/about.html.erb]: CRITICAL: Love the layout/concept but don't approve of my picture being usedon the site. :(

File: spec/features/attending_events_spec.rb [hunk 4]
```diff
@@ -42,6 +42,7 @@
     visit events_path
     find_link("Blessings wedding").trigger("click")
     find_link("Attend this event").trigger("click")
+
     within ".modal-content" do
       page.execute_script("$('#ticket_type_1').prop('checked', true)")
       fill_in "tickets_quantity_1", with: 1
```
Comment (andela-megwali) [inline spec/features/attending_events_spec.rb]: Overall, great job. Well done. 👍

**Comment by andela-madegbola** [comment]
Nice job, Chineze! You have a good eye for design. It is simple and clean.

**Comment by andela-ajamiu** [comment]
Great job. Well-done!!!

File: spec/features/attending_events_spec.rb (hunk 4 above)
Comment (andela-cnwosu) [inline spec/features/attending_events_spec.rb]: Thanks, Mic. I will implement your comments as soon as possible

**Comment by andela-kadeniyi** [comment]
@andela-cnwosu @andela-mogala Great job guys. Will merge this.

**Pull request merged**